dracut/module-setup.sh

```diff
@@ -50,6 +50,7 @@ install() {
     inst_hook pre-pivot 91 "$moddir/anaconda-copy-prefixdevname.sh"
     inst_hook pre-pivot 95 "$moddir/anaconda-set-kernel-hung-timeout.sh"
     inst_hook pre-pivot 99 "$moddir/save-initramfs.sh"
+    inst_hook cleanup 99 "$moddir/nfsrepo-cleanup.sh"
```

Comment:
Maybe I'd use rather
`inst_hook cleanup 98 "$moddir/anaconda-nfsrepo-cleanup.sh"`
because it is anaconda specific hook and there is also
`99-nfsroot-cleanup.sh` (https://github.com/dracutdevs/dracut/blob/master/modules.d/95nfs/nfsroot-cleanup.sh) hook in installer initramfs, and I think anaconda hook should precede it explicitly based on the number (rather than alphabetical order)

Reply:
Thanks Radek, I changed the file name into anaconda-nfsrepo-cleanup.sh.

Reply:
Don't we need to modify also the line above to:
inst_hook cleanup 98 "$moddir/anaconda-nfsrepo-cleanup.sh" ?

Reply:
Oops, I just changed it as well. Thanks!

Reply:
Thank you for the fix-up.I was also proposing a change of the priority of the script (99->98 .. see my first comment in this thread) - if you agree please update it as well.

Also, could you squash all the commits in this PR into a single one so it is easier to port.

Reply:
Hi Radek, 
Sorry, I missed the number change. I updated it just now. Also, I squashed commits into a single one. 
Hopefully, I could do that correctly. Thanks!